app/resources/forms.py

```diff
@@ -0,0 +1,40 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    IntegerField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length, URL
+
+
+class ResourceForm(Form):
+    address_autocomplete = StringField('Enter the address')
+    name = StringField('Name', validators=[
+        InputRequired(),
+        Length(1, 64)
+    ])
+    description = StringField('Description', validators=[
+        InputRequired(),
+    ])
+    website = StringField('Website', validators=[
+        URL()
```

Comment:
I don't think this field should be mandatory, and I think by putting this validator here we make it "mandatory". I believe including the `Optional()` validator will make it not mandatory.